Get_Repo_Details/get_repo_details.py

```diff
@@ -0,0 +1,49 @@
+# Script to GET Repo details from user
+# @Author - Sandip Dutta
+# Email - duttasandip11100@gmail.com
+
+# IMPORTS
+import json
+import requests
```

Comment:
Have this in a single script. At this stage, there is no need for modularization

Reply:
Yes. Converted this into a single script